webapp/graphite/metric_filter/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.contrib.auth.models import Group
+from django.contrib.auth.models import User
+from django.core.exceptions import ObjectDoesNotExist
+
+class MetricFilter(models.Model):
+  id = models.AutoField(primary_key=True)
+  filter = models.CharField(max_length=256)
+  description = models.CharField(max_length=200)
+  type_filter = models.IntegerField(choices=[(0, 'include'), (1, 'exclude')])
+  __str__ = lambda self: "[%s]: %s " % (getTypeFilter(self.type_filter) ,self.filter)
+
+  class Meta:
+    db_table = "metric_filter"
+
+class UserMetricFilter(models.Model):
+  user = models.ForeignKey(User)
```

Comment:
You can use [related_name](https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.ForeignKey.related_name) and do `user.metric_filters.all()` instead of having to define a method for getting filters for a given user.